.github/workflows/prod.yml

```
@@ -0,0 +1,47 @@
+        uses: azure/container-apps-deploy-action@v2
+        with:
+          appSourcePath: ${{ github.workspace }}
+          dockerfilePath: /
+          registryUrl: ${{ secrets.REGISTRY_URL }}
+          registryUsername: ${{ secrets.REGISTRY_USERNAME }}
+          registryPassword: ${{ secrets.REGISTRY_PASSWORD }}
+          containerAppName: ${{ secrets.CONTAINER_APP_NAME }}
+          resourceGroup: ${{ secrets.RESOURCE_GROUP }}
+          imageToBuild: ${{ secrets.REGISTRY_URL }}/${{ secrets.CONTAINER_APP_NAME }}:${{ github.run_number }}
+          environmentVariables: "NEXT_PUBLIC_BASE_URL=${{ secrets.NEXT_PUBLIC_BASE_URL }} NEXT_PUBLIC_SANITY_PROJECT_ID=${{ secrets.NEXT_PUBLIC_SANITY_PROJECT_ID }} NEXT_PUBLIC_SANITY_DATASET=${{ secrets.NEXT_PUBLIC_SANITY_DATASET }} SANITY_API_READ_TOKEN=${{ secrets.SANITY_API_READ_TOKEN }} "
+          buildArguments: "NEXT_PUBLIC_BASE_URL=${{ secrets.NEXT_PUBLIC_BASE_URL }} NEXT_PUBLIC_SANITY_PROJECT_ID=${{ secrets.NEXT_PUBLIC_SANITY_PROJECT_ID }} NEXT_PUBLIC_SANITY_DATASET=${{ secrets.NEXT_PUBLIC_SANITY_DATASET }} SANITY_API_READ_TOKEN=${{ secrets.SANITY_API_READ_TOKEN }} "
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)